forecasttools/pull_data_cdc_gov.py

```diff
@@ -0,0 +1,145 @@
+from datetime import date
+
+import polars as pl
+from cfasodapy import Query
+
+# Dataset IDs and metadata for commonly
+# used (within STF) data.cdc.gov datasets
+data_cdc_gov_datasets = pl.DataFrame(
+    {
+        "key": ["nhsn_hrd_prelim", "nhsn_hrd_final", "nssp_prop_ed_visits"],
+        "id": ["mpgq-jmmr", "ua7e-t2fy", "rdmq-nq56"],
+        "date_column": ["weekendingdate", "weekendingdate", "week_end"],
+        "location_column": ["jurisdiction", "jurisdiction", "county"],
+    }
+)
+
+
+def get_dataset_info(dataset_key: str) -> dict:
+    """Look up dataset information by key."""
+    filtered = data_cdc_gov_datasets.filter(pl.col("key") == dataset_key)
+    if filtered.height == 0:
+        raise ValueError(
+            f"Dataset key '{dataset_key}' not found in dataset table"
+        )
+    return filtered.to_dicts()[0]
```

Comment:
```suggestion
    return filtered.row(0, named=True)
```
Seems maybe more idiomatic, but up to you.